replace atlas.pt? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
(EOF or read error, treating as "[N]one" ...)

PackagesNotFoundError: The following packages are not available from current channels:

  - antspy

Current channels:

  - https://conda.anaconda.org/conda-forge
  - https://repo.anaconda.com/pkgs/main
  - https://repo.anaconda.com/pkgs/r

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.


Traceback (most recent call last):
  File "Data_generating.py", line 153, in <module>
    images_pet, images_mri, labels = generate_data_path_less()
  File "Data_generating.py", line 84, in generate_data_path_less
    df = pd.read_csv(file)
  File "/home/l.peiwang/miniconda3/envs/preprocess/lib/python3.7/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/l.peiwang/miniconda3/envs/preprocess/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/l.peiwang/miniconda3/envs/preprocess/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/l.peiwang/miniconda3/envs/preprocess/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/l.peiwang/miniconda3/envs/preprocess/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/l.peiwang/miniconda3/envs/preprocess/lib/python3.7/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 51, in __init__
    self._open_handles(src, kwds)
  File "/home/l.peiwang/miniconda3/envs/preprocess/lib/python3.7/site-packages/pandas/io/parsers/base_parser.py", line 229, in _open_handles
    errors=kwds.get("encoding_errors", "strict"),
  File "/home/l.peiwang/miniconda3/envs/preprocess/lib/python3.7/site-packages/pandas/io/common.py", line 707, in get_handle
    newline="",
PermissionError: [Errno 13] Permission denied: '/scratch/jjlee/Singularity/ADNI/bids/derivatives/table_preclinical_cross-sectional.csv'
